bentoml/tracing/__init__.py

```diff
@@ -27,23 +28,27 @@
 @lru_cache(maxsize=1)
 def get_tracer(
     tracer_type=Provide[BentoMLContainer.config.tracing.type],
-    zipkin_server_url: str = Provide[BentoMLContainer.config.tracing.zipkin.url],
-    jaeger_server_address: str = Provide[
+    zipkin_server_url: Optional[str] = Provide[
+        BentoMLContainer.config.tracing.zipkin.url
+    ],
+    jaeger_server_address: Optional[str] = Provide[
         BentoMLContainer.config.tracing.jaeger.address
     ],
-    jaeger_server_port: str = Provide[BentoMLContainer.config.tracing.jaeger.port],
+    jaeger_server_port: Optional[str] = Provide[
+        BentoMLContainer.config.tracing.jaeger.port
+    ],
 ):
     # isinstance check here allow trace to be used where the top-level entry point has
```

Comment:
I guess we don't need this part anymore?

Reply:
Yeah. Here is not the only place that we do not need anymore after introducing simple-di. I'll have a full cleaning after this PR.